src/utils/keygen.rs

```diff
@@ -72,18 +74,37 @@ impl Keypair {
     pub fn to_pkcs8_pem(&self) -> Result<String, Error> {
         self.repr
             .key
-            .to_pkcs8_pem(LineEnding::default())
+            .to_pkcs8_pem(LineEnding::LF)
             .map_err(|err| {
                 tracing::error!("Failed to convert signing key to PEM: {err:?}");
                 Error::PemGenFailed
             })
-            .map(|pem| pem.to_string())
+            .map(|pem| pem.to_string().trim().to_string())
```

Comment:
Does this work? I believe there is a shorter syntax.
```suggestion
            .map(|pem| pem.trim().to_owned())
```